frontend/src/learning/components/ItemDisplay/ProgressBar.js

```diff
@@ -0,0 +1,138 @@
+import React, {Component} from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
+
+// Body: use padding to adjust Bar's position 
+const Body = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 10px 30px;
+`
+// Outer bar container
+const BarContainer = styled.div`
+  background-color: #d8d8d8;
+	position: relative;
+  height: 7px;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+`
+// Inner color bar
+const Bar = styled.div`
+  background: #d8d8d8;
+  box-shadow: 2px 2px 5px #d8d8d8;
+  display: flex;
+  align-items: left;
+  justify-content: left;
+  height: 100%;
+  width: 0px;
+  opacity: 1;
+  transition: 1s ease 0.3s;
+  `
+  // Text Container Below
+  const TextContainer = styled.div`
+    position: relative;
+    height: 10px;
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    margin-top: 7px;
+    font-size: 15px;
+    
+`
+const Text = styled.div`
+  width: 0px;
+  min-width: 70px;
+  text-align: left;
+`
+
+export default class ProgressBar extends React.Component {
+
+  static propTypes = {
+    mastered: PropTypes.number,
+    familiar: PropTypes.number,
+    remaining: PropTypes.number,
+    bonus: PropTypes.number,
+  }
+
+  constructor(props){
+    super(props)
+
+    this.total = this.props.mastered + this.props.familiar + this.props.remaining + this.props.bonus
+
+    this.bar = {
+        bar1: {
+          width: `${this.props.mastered / this.total * 100}%`,
+          background: "rgb(85, 162, 30)",
```

Comment:
All strings should be quoted with single quotes, just like you did for the imports.